hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoGetRecordsRegression.java

```diff
@@ -122,18 +124,20 @@ private HapiSpec failsForUnfundablePayment() {
                         .hasAnswerOnlyPrecheck(INSUFFICIENT_PAYER_BALANCE));
     }
 
+    @HapiTest
     private HapiSpec failsForInsufficientPayment() {
         return defaultHapiSpec("FailsForInsufficientPayment")
-                .given()
+                .given(cryptoCreate(PAYER))
                 .when()
-                .then(getAccountRecords(GENESIS).nodePayment(1L).hasAnswerOnlyPrecheck(INSUFFICIENT_TX_FEE));
+                .then(getAccountRecords(GENESIS).payingWith(PAYER).nodePayment(1L).hasAnswerOnlyPrecheck(INSUFFICIENT_TX_FEE));
     }
 
-    private HapiSpec failsForMissingPayment() {
-        return defaultHapiSpec("FailsForMissingPayment")
+    @HapiTest
+    private HapiSpec failsForInvalidTrxBody() {
+        return defaultHapiSpec("failsForInvalidTrxBody")
                 .given()
                 .when()
-                .then(getAccountRecords(GENESIS).useEmptyTxnAsAnswerPayment().hasAnswerOnlyPrecheck(NOT_SUPPORTED));
+                .then(getAccountRecords(GENESIS).useEmptyTxnAsAnswerPayment().hasAnswerOnlyPrecheck(INVALID_TRANSACTION_BODY));
```

Comment:
Does this also pass in mono service? Curious why the change here

Reply:
It was never working in Mono. It was disabled

Reply:
For tests that don't pass for mono either, I've been putting a comment above the test with `FUTURE` to come back to it, and figure out later why it isn't passing in mono. I don't know that we want to change any of these tests just yet. @Neeharika-Sompalli can you confirm?

Reply:
We fixed it with @tinker-michaelj